src/objects/Data_Import_Settings__c.object

```diff
@@ -117,6 +152,18 @@
         <type>Text</type>
         <unique>false</unique>
     </fields>
+    <fields>
+        <fullName>Field_Mapping_Method__c</fullName>
+        <description>Defines the way fields on the NPSP Data Import record are mapped to other objects in Salesforce. The options are Help Text or Data Import Advanced Mapping.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>Defines the way fields on the NPSP Data Import record are mapped to other objects in Salesforce. The options are Help Text or Data Import Advanced Mapping.</inlineHelpText>
+        <label>Field Mapping Method</label>
+        <length>40</length>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
```

Comment:
Do we need code in the package install script to set any of these for orgs that are receiving a push upgrade of the package?

Reply:
Nope! Our deployment and rollout plan does not rely on this.